dts/xtensa/intel/intel_adsp_cavs25_tgph.dtsi

```diff
@@ -135,5 +179,220 @@
 			reg = <0x3000 0x1000>;
 			paddr-size = <11>;
 		};
+
+		ssp0: ssp@77000 {
```

Comment:
try and keep this in address sorted order

Reply:
@galak Updated. I did not fix the existing nodes, which already seem to out-of-order. Plus if we start moving these around, we should change also intel_adsp_cavs25.dtsi as it is very convenient to be able to "diff" the cavs25 variants.